<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>60</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tajawal ExtraBold</family>
     <pointsize>32</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(190, 141, 109);
font: 81 32pt &quot;Tajawal ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>CART</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tajawal ExtraBold</family>
     <pointsize>32</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 81 32pt &quot;Tajawal ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>THE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            width: 30px;
			height:30px;
            border: none;
			background-color: transparent; 
           border-radius: 15px;
}
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Dr. Rod/Desktop/Year2/python/image/no.png</normaloff>C:/Users/Dr. Rod/Desktop/Year2/python/image/no.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="output_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
            width: 30px;
			height:30px;
            border: none;
           border-radius: 15px;
	background-color: none;
  }
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Dr. Rod/Desktop/Year2/python/image/back (1).png</normaloff>C:/Users/Dr. Rod/Desktop/Year2/python/image/back (1).png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>1211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(232, 174, 133,0.4);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>1211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(232, 174, 133,0.4);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>1211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(232, 174, 133,0.4);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>330</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/Picsart_25-01-31_21-51-30-608.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/Picsart_25-01-31_19-58-52-164.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>121</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/Picsart_25-01-31_21-58-22-673.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans ExtraBold</family>
     <pointsize>16</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: none;
font: 81 16pt &quot;Open Sans ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>Tasy And Healthy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
background-color:none;
</string>
   </property>
   <property name="text">
    <string>$15.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
background-color:none;
</string>
   </property>
   <property name="text">
    <string>$15.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>480</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans ExtraBold</family>
     <pointsize>16</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: none;
font: 81 16pt &quot;Open Sans ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>Tasy And Healthy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
background-color:none;
</string>
   </property>
   <property name="text">
    <string>$15.99</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans ExtraBold</family>
     <pointsize>16</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: none;
font: 81 16pt &quot;Open Sans ExtraBold&quot;;</string>
   </property>
   <property name="text">
    <string>Tasy And Healthy</string>
   </property>
  </widget>
  <widget class="QLabel" name="socurelabel1">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>220</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
            border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string>     0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addbutton1">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>220</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            width: 30px;
			height:30px;
            border: none;
			background-color: black; 
           border-radius: 10px;
font: 75 18pt &quot;MS Shell Dlg 2&quot;;

color:white;
}
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="subbutton1">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>220</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            width: 30px;
			height:30px;
            border: none;
			background-color: black; 
           border-radius: 10px;
font: 75 18pt &quot;MS Shell Dlg 2&quot;;

color:white;
}
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletebutton1">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>230</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            width: 30px;
			height:30px;
            border: none;
			background-color: transparent; 
           border-radius: 15px;
}
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../image/x.png</normaloff>../image/x.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="deletebutton2">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>360</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            width: 30px;
			height:30px;
            border: none;
			background-color: transparent; 
           border-radius: 15px;
}
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../image/x.png</normaloff>../image/x.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="subbutton2">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>350</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            width: 30px;
			height:30px;
            border: none;
			background-color: black; 
           border-radius: 10px;
font: 75 18pt &quot;MS Shell Dlg 2&quot;;

color:white;
}
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addbutton2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>350</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            width: 30px;
			height:30px;
            border: none;
			background-color: black; 
           border-radius: 10px;
font: 75 18pt &quot;MS Shell Dlg 2&quot;;

color:white;
}
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}

</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="socurelabel2">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>350</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
            border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string>     0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletebutton3">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>500</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            width: 30px;
			height:30px;
            border: none;
			background-color: transparent; 
           border-radius: 15px;
}
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../image/x.png</normaloff>../image/x.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="subbutton3">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>490</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            width: 30px;
			height:30px;
            border: none;
			background-color: black; 
           border-radius: 10px;
font: 75 18pt &quot;MS Shell Dlg 2&quot;;

color:white;
}
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addbutton3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>490</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            width: 30px;
			height:30px;
            border: none;
			background-color: black; 
           border-radius: 10px;
font: 75 18pt &quot;MS Shell Dlg 2&quot;;

color:white;
}
QPushButton:hover {
background-color: rgba(232, 174, 133,0.8);
}
</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="socurelabel3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>490</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
            border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string>     0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
